src/components/sound.js

```diff
@@ -155,7 +156,7 @@ module.exports.Component = registerComponent('sound', {
     this.pool.children.forEach(function (sound) {
       sound.onEnded = function () {
         sound.isPlaying = false;
-        el.emit('sound-ended', {index: i});
+        el.emit('sound-ended', {index: i, id: self.evtDetail && self.evtDetail.id, name: self.evtDetail && self.evtDetail.name});
```

Comment:
Where is `evtDetail` defined?

Reply:
It's an attribute that all components have. https://github.com/aframevr/aframe/blob/master/src/core/component.js#L38